index.html

```diff
@@ -44,7 +44,7 @@
                     <a class="dropdown-item" href="pages/ccf-anatomical-structures.html">CCF Anatomical Structures, Cell
                         Types and Biomarkers (ASCT+B) Tables</a>
                     <a class="dropdown-item" href="pages/ccf-3d-reference-library.html">CCF 3D Reference Object Library</a>
-                    <a class="dropdown-item" href="pages/ccf-2d-reference-library.html">CCF 2D Reference FTU Library</a>
+                    <a class="dropdown-item" href="pages/ccf-2d-reference-library.html">CCF 2D Reference Functional Tissue Unit (FTU) Library</a>

```

Comment:
@Medina88 Please check all .html pages (especially in the pages directory) and update them there as well. You'll see this menu is copied in all subpages.